projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -315,30 +403,63 @@ export function buildPromptForBlock( {
 	} );
 }
 
-type GetPromptOptionsProps = {
-	content?: string;
-	contentType?: 'generated' | string;
-	tone?: ToneProp;
-	language?: string;
-};
-
 /**
  * Returns a prompt based on the type and options
  *
  * @param {PromptTypeProp} type           - The type of prompt.
  * @param {GetPromptOptionsProps} options - The prompt options.
  * @returns {Array< PromptItemProps >}      The prompt.
  */
-export function buildPromptForExtensions(
+export function getPrompt(
```

Comment:
This function is only used for extensions and is independent from all the other functions. `getPrompt` in this case seems misleading. Maybe this should be moved to `/extensions/ai-assistant/lib`?

Reply:
yes, let me do it in a follow-up please